cfme/intelligence/reports/reports.py

```diff
@@ -546,8 +546,9 @@ def delete(self, cancel=False):
             view.flash.assert_no_error()
         else:
             view.flash.assert_no_error()
-            # TODO Doesn't work due to this BZ https://bugzilla.redhat.com/show_bug.cgi?id=1489387
-            # view.flash.assert_message("Successfully deleted Saved Report from the CFME Database")
+            view.flash.assert_message(
+                'Report "{}": Delete successful'.format(self.report.menu_name)
+            )
 
```

Comment:
@valaparthvi 
According to me, It would be great, if we move ```view.flash.assert_message``` to tests. And keep ```view.flash.assert_no_error()``` here only.